lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -1809,17 +1809,122 @@ bool SwiftLanguageRuntimeImpl::IsValidErrorValue(ValueObject &in_value) {
   return true;
 }
 
+static bool
+CheckIfAnyTypesDeclaredInLLDB(const swift::reflection::TypeRef *typeref) {
+  using swift::Demangle::Node;
+
+  const auto *protocol_typeref = llvm::dyn_cast_or_null<
+      const swift::reflection::ProtocolCompositionTypeRef>(typeref);
+
+  assert(protocol_typeref);
+  if (!protocol_typeref) {
+    assert(false);
```

Comment:
this is redundant with line 1819

Reply:
Sorry, I should've removed these asserts.